cli/cli_test.go

```diff
@@ -93,3 +93,9 @@ func TestDumpMissingFile(t *testing.T) {
 	assert.Equal(t, expected, testTerminal.ErrorBuf.String())
 	assert.Empty(t, testTerminal.OutputBuf.Bytes())
 }
+
+func TestConnect(t *testing.T) {
+	args := []string{"connect", "--verbose", "squareup.com:443"}
```

Comment:
We should spin up a server in the test and connect to it, rather than relying on an outside source.

Maybe use https://golang.org/pkg/net/http/httptest/#Server

Reply:
makes sense